### PR TITLE
Bugfixes

### PR DESCRIPTION
**Pull request by cuevaskoch**
Updated proptypes and references.

### CODE REVIEW

File: Konami.js [hunk 1]
```diff
@@ -32,13 +32,10 @@ class Konami extends Component {
 
 Konami.propTypes = {
   easterEgg: PropTypes.func.isRequired,
-  konami: PropTypes.string
+  konami: PropTypes.arrayOf(PropTypes.number)
```
Comment (cuevaskoch) [inline Konami.js:35]: I updated the `propTypes`to match what was listed in `defaultProps`

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-konami",
-  "version": "0.2.2",
+  "version": "0.2.3",
```
Comment (cuevaskoch) [inline package.json:3]: This is my first github PR so I'm unsure of etiquette.. I bumped the version number?  Should I not do that?

File: README.md [hunk 3]
```diff
@@ -12,4 +12,4 @@ Import the component, render it and you're all set
 | Type        | Key                 | Description
 |------------ |---------------------|---------------------------------------------------------
 | (func)      | `easterEgg`         | what happens when you enter the konami code (optional)
-| (array)     | `konami`            | you can change the key combination (array of strings) (optional)
+| (array)     | `konami`            | you can change the key combination (array of ascii keycodes) (optional)
```
Comment (cuevaskoch) [inline README.md:15]: I updated documentation to match `defaultProps` as set in the component

File: README.md [hunk 4]
```diff
@@ -12,4 +12,4 @@ Import the component, render it and you're all set
 | Type        | Key                 | Description
 |------------ |---------------------|---------------------------------------------------------
 | (func)      | `easterEgg`         | what happens when you enter the konami code (optional)
```
Comment (FloValence) [inline README.md:14]: As we remove the defaultProp `easterEgg` we should mark it as Required

File: package.json (hunk 2 above)
Comment (FloValence) [reply package.json:3]: Yes, it's a patch :-)

**Review by FloValence — APPROVED** [review]
Thank you ! I did the update in the haste.

The built file is missing, but I'll add it: you should be able to build using `npm run build` => I should add this in the README too ^^

**Pull request merged**